.docker/ubuntu-22.04/Dockerfile

```diff
@@ -4,7 +4,9 @@ LABEL org.opencontainers.image.source https://github.com/microsoft/msquic
 
 ENV DEBIAN_FRONTEND=noninteractive
 
-RUN apt-get update && apt-get install --no-install-recommends -y apt-transport-https \
+RUN apt-get update && \
+    apt-get install -y gcc-multilib && \
```

Comment:
why do we have a separate line for installing `gcc-multilib`?

Reply:
My VM could install gcc-multilib before installing rest of packages for some reason...
changing experiment environment to container